gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Thu May 18 20:58:24 PDT 2017
+#Mon Oct 22 23:06:07 EEST 2018
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-3.3-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.6-all.zip
```

Comment:
Grrr(((

Reply:
In Gradle 4.6 were fixed some critical bugs, it also supports the latest version of different libs and frameworks.
Moreover, it is an important part of your environment, so I prefer to keep it up to date. 
Anyway, Gradle just a build tool and you won`t differ any changes from your side.

Reply:
You shouldn't upgrade your environment up to date, just because of new version. It can bring new bugs or cause other problems in your soft. But fixing critical bugs is a good reason for upgrading.